app/js/containers/Buyer/Listing.scss

```diff
@@ -0,0 +1,32 @@
+@import "../../../../node_modules/bootstrap/scss/functions";
+@import "../../../../node_modules/bootstrap/scss/variables";
+
+$logo-size: 64px;
+.logo-circle {
```

Comment:
If it is not wrapped around container name I feel this need to be extracted as a component